### PR TITLE
Add first Behat scenario for API

### PR DESCRIPTION
**Pull request by Roshyo**

### CODE REVIEW

File: tests/Behat/Resources/api/clients.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://symfony.com/schema/dic/services"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true"/>
```
Comment (loevgaard) [inline tests/Behat/Resources/api/clients.xml]: Does this needs to be public?

File: tests/Behat/Resources/context/api.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://symfony.com/schema/dic/services"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true"/>
```
Comment (loevgaard) [inline tests/Behat/Resources/context/api.xml:6]: Does this needs to be public?

File: tests/Behat/Resources/context/setup.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://symfony.com/schema/dic/services"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true"/>
```
Comment (loevgaard) [inline tests/Behat/Resources/context/setup.xml:6]: Does this needs to be public?

**Review by loevgaard — CHANGES_REQUESTED** [review]
Can you check which services needs to be public? I am pretty sure it's only the pages or something

**Comment by Roshyo** [comment]
Indeed, only the Contexts need to be public. Pages and api client can be private

**Review by loevgaard — APPROVED** [review]

**Pull request merged**